lib/Optimizer/TrainingPreparation.cpp

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Optimizer/TrainingPreparation.h"
+#include "glow/Graph/PlaceholderBindings.h"
+
+namespace glow {
+
+TensorInitializer getDefaultTensorInitializer() {
+  return [](Function *F, Node *node, NodeValue &input, Tensor *tensor) {
```

Comment:
Nit: This closure does not capture anything. It could be expressed as a static free-standing function before `getDefaultTensorInitializer ` and `getDefaultTensorInitializer` would just return it.